tests/libcxx/run_tests_locally.sh

```diff
@@ -24,7 +24,8 @@ while read test; do
     FAILED=$(echo "$OUTPUT" | grep failed)
     FAIL_ENCLAVE=$(echo "$OUTPUT" | grep OE_ENCLAVE_ABORTING)
     TIMED_OUT=$(echo "$OUTPUT" | grep -F '[one] Terminated')
-    if [[ -z $FAILED && -z $FAIL_ENCLAVE && -z $TIMED_OUT ]]  
+    TERMINATED=$(echo "$OUTPUT" | grep -F 'terminate called')
+    if [[ -z $FAILED && -z $FAIL_ENCLAVE && -z $TIMED_OUT && -z $TERMINATED ]]  
```

Comment:
Please describe the usage of this script in libcxx/README.

Reply:
Missed that! Thanks, I've added it now